forge-gui/res/cardsfolder/upcoming/bill_potts.txt

```diff
@@ -0,0 +1,9 @@
+Name:Bill Potts
+ManaCost:3 R
+Types:Legendary Creature Human
+PT:2/4
+T:Mode$ SpellCast | ValidCard$ Instant.YouCtrl,Sorcery.YouCtrl,Activated.YouCtrl | ValidActivatingPlayer$ You | IsSingleTarget$ True | TargetsValid$ Card.Self | ActivationLimit$ 1 | Execute$ TrigCopy | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast an instant or sorcery spell that targets only Bill Potts or activate an ability that targets only Bill Potts, copy that spell or ability. You may choose new targets for the copy. This ability triggers only once each turn.
```

Comment:
wrong triggertype, use `SpellAbilityCast` and you also need to filter better, your ValidCard is using a SpellAbility property so that can't work